src/styles/2-components/_header.scss

```diff
@@ -0,0 +1,190 @@
+.header {
+  background-image: url("../images/header/bg-mobile-2x.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  box-sizing: border-box;
+  min-height: 100vh;
+
+  padding-bottom: 40px;
+
+  @media (min-width: 500px) {
+    background-image: url("../images/header/bg-tablet.jpg");
+  }
+
+  @media (min-width: 1070px) {
+    background-image: url("../images/header/bg-desktop-x4.jpg");
+  }
+
+  &__grid-wrapper {
+    @extend %pageGrid;
+    align-items: center;
+    grid-template-rows: 1fr calc(100vh - 184px) 1fr;
+
+    @include onTablet {
+      grid-template-rows: 1fr calc(100vh - 264px) 1fr;
+    }
+  }
+
+  &__top {
+    padding-block: 24px;
+    @include flex-align(space-between);
+  }
+
+  &__top-logo-link {
+    display: block;
+  }
+
+  &__icons-box {
+    grid-column: 2 / 3;
+    text-align: right;
+
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 24px;
+
+    position: relative;
+
+    &-number {
+      font-family: $addition-font-family-1, sans-serif;
+      font-size: 12px;
+      font-weight: 700;
+      width: max-content;
+      letter-spacing: 2px;
+      position: absolute;
+      right: 100px;
+      top: 50%;
+      transform: translateY(-50%);
+
+      overflow-x: hidden;
+      box-sizing: border-box;
```

Comment:
It is better to have box-sizing set globally via `*`, unless you have a reason to